<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>findcus</class>
 <widget class="QDialog" name="findcus">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>770</width>
    <height>489</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>770</width>
    <height>489</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>770</width>
    <height>489</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>查询顾客信息</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>0</y>
     <width>258</width>
     <height>45</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 26pt &quot;方正舒体&quot;;</string>
   </property>
   <property name="text">
    <string>查询顾客信息</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>101</y>
     <width>751</width>
     <height>381</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>741</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,0,6">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QComboBox" name="comboBox">
        <item>
         <property name="text">
          <string>请选择查询方式</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>按单号查询</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>按编号查询</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>按姓名查询</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>按性别查询</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>按年龄查询</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>按住址查询</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>按电话查询</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>按症状查询</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>按已购药品查询</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>按经办人查询</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>按时间查询</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>按备注查询</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QComboBox" name="cb_fuhao">
      <item>
       <property name="text">
        <string>=</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>&gt;</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>&lt;</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>&gt;=</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>&lt;=</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QDateEdit" name="dateEdit"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>查询</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
